django/core/checks/settings.py

```diff
@@ -0,0 +1,122 @@
+from django.conf import settings
+
+from . import Tags, Warning, register
+
+DEPRECATED_SETTINGS = {
```

Comment:
From the discussion, I had pictured this as listing **removed settings**, rather than merely deprecated ones, which would presumably have their own checks. 🤔

Reply:
You are totally right. I mixed it up and will fix 👌